bower.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "n-image",
-  "version": "0.0.0",
```

Comment:
Am getting this error when running `nbt bottle` while listing a version:-

`bower.json should not contain a version property; versioning is done with git tags`

Have looked at other bower components (all of those required by `next-front-page`) and none of the `bower.json` files for those contain a version number.